.gitlab/ci/bucket-upload.yml

```diff
@@ -456,3 +457,17 @@ slack-notify-bucket-upload:
   extends:
     - .trigger-slack-notification
     - .bucket-upload-rule-always
+
+
+upload-id-set-bucket:
+  tags:
+    - gke
+  stage: prepare-testing-bucket
+  extends:
+    - .bucket-upload-rule
+    - .default-job-settings
+  script:
+    # This is needed because we still use id_set.json in other repos
+    - !reference [.create-id-set-xsoar]
+    - gcloud auth activate-service-account --key-file="$GCS_MARKET_KEY"
+    - gsutil cp $ARTIFACTS_FOLDER/id_set.json "gs://$GCS_MARKET_BUCKET/content/id_set.json"
```

Comment:
Currently we are updating the id-set in the bucket in the script `Tests/scripts/prepare_content_packs_for_testing.sh`
I think that if we are going to update the id-set in this job then you can remove it there